modules/ocf/manifests/etc.pp

```diff
@@ -0,0 +1,10 @@
+class ocf::etc {
+  file { '/etc/ocf':
+    ensure  => directory,
+    source  => 'puppet:///etc',
+    owner   => root,
+    group   => root,
+    mode    => '0755',
+    recurse => true,
```

Comment:
I'd add `purge => true` on this too so that unmanaged files are deleted, since they wouldn't be with this current setup.